src/elements/content-sidebar/skills/transcript/__tests__/__snapshots__/EditingTranscriptRow.test.js.snap

```diff
@@ -12,12 +12,8 @@ exports[`elements/content-sidebar/Skills/Transcript/TranscriptRow should correct
   <div
     className="be-transcript-text"
   >
-    <TextareaAutosize
-      inputRef={[Function]}
+    <ForwardRef(TextareaAutosize)
       maxRows={10}
-      onChange={[Function]}
-      onHeightChange={[Function]}
-      useCacheForDOMMeasurements={false}
```

Comment:
I don't fully follow, why are the props disappearing here? Any chance the forward ref isn't passing the props to `TextareaAutosize` correctly? If you manually confirmed, that's fine with me

Reply:
if you look at the test, the props that are being removed in the snapshot, are not provided. `maxRows` remains because the test is providing it

Reply:
hmm that's weird, if they werent provided why did they show up in the first place? Have you confirmed it still looks normal?

Reply:
v7 had static defaults, v8 converted TS, so no static defaults